Toolbar/Toolbar.jsx

```diff
@@ -62,9 +62,12 @@ class ToolbarSection extends MaterialComponent {
 }
 
 class ToolbarIcon extends MaterialComponent {
-  constructor() {
+  constructor(props) {
     super();
     this.componentName = "toolbar__icon";
+    if (props.menu) {
```

Comment:
The prop is used for the left-most icon. For other icons the `--menu` shouldn't be used. I didn't make a new component but rather reused the old one, just added a prop. If you want I could make a separate component instead (I think that goes against DRY though).